index.html

```diff
@@ -16,34 +17,108 @@ <h1 class="heading">IdeaBox</h1>
           <button class="starred-ideas-button">Show Starred Ideas</button>
         </div>
       </section>
-      <form class="container-top">
-        <div class="title-area">
-          <label for="title">Title</label>
-          <input type="text" id="userTitle" value="title" required>
-        </div>
-        <div class="body-area">
-          <label for="body">Body</label>
-          <input type="text" id="userBody" value="body" required>
-        </div>
-        <button class="save-button">Save</button>
-        <div class="search-ideas-box">
-          <img src="assets/search.svg" alt="search button">
-          <input class="search-field" type="text" id="searchField" value="search" required placeholder="Search Ideas">
-        </div>
-      </form>
-      <section class="container-bottom">
-        <article class="idea-card">
```

Comment:
Retained semantic naming to element tags which is helpful. We'll be using this in JS ahead